frontend/package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
```

Comment:
just curious - did you add this on purpose?

Reply:
I don't recall touching these files. Should I fix them?

Reply:
nah it's not a big deal! See this [post](https://stackoverflow.com/a/22345808) if you're curious what it means